salt/cloud/clouds/azurearm.py

```diff
@@ -1296,7 +1342,7 @@ def _query_node_data(name, bootstrap_interface):
         return ip_address
 
     try:
-        data = salt.utils.cloud.wait_for_ip(
+        data = __utils__["cloud.wait_for_ip"](
```

Comment:
Why are you using `__utils__` instead of how it was?

Reply:
salt.utils. causeing issue with the loader